grails-app/views/stockMovement/_receipts.gsp

```diff
@@ -48,7 +48,7 @@
                     ${receiptItem?.binLocation?.name}
                 </td>
                 <td>
-                    ${receiptItem?.quantityReceived?:0}
+                    ${receiptItem?.getReceivedQuantity()?:0}
```

Comment:
... instead, add the conditional around the view.
```
<g:if test="${receipt.receiptStatusCode == ReceiptStatusCode.RECEIVED}">
${receiptItem?.quantityReceived?:0}
</g:if>
<g:else>
<g:message code="default.tbd.label" default="TBD"/>
</g:else>
```

Also, in general, there's no need for the get in getReceivedQuantity. The following are equivalent.
```
${receiptItem?.getReceivedQuantity()?:0}
```
and 
${receiptItem?.receivedQuantity?:0}

Reply:
For the "TBD" part, in the ticket ask Jesse what she wants there. Maybe show a screenshot of what it looks like and ask if she'd prefer a blank cell, 0, or something like TBD, which is short for "To be determined".

Reply:
@jmiranda I've added changed commit, could you review?